impl/src/test/java/org/ehcache/impl/internal/store/heap/OnHeapStoreProviderTest.java

```diff
@@ -83,7 +83,7 @@ public void testRankCachingTier() throws Exception {
   private void assertRank(final Store.Provider provider, final int expectedRank, final ResourceType<?>... resources) {
     assertThat(provider.rank(
       new HashSet<>(Arrays.asList(resources)),
-        Collections.<ServiceConfiguration<?>>emptyList()),
+        Collections.<ServiceConfiguration<?, ?>>emptyList()),
```

Comment:
I believe you can elide the generic parameter here.

Reply:
Done